packages/shared-component--membershipHero/contentful/membershipHero.js

```diff
@@ -0,0 +1,150 @@
+module.exports = function (migration) {
+    const hero = migration.createContentType('membershipHero');
+
+    hero.name('-- Membership hero')
+        .description('A page header that can be used to promote membership.')
+
+    hero.createField('name')
+        .name('Name')
+        .type('Symbol')
+        .required(true)
+
+    hero.createField('homeHero')
+        .name('Main homepage hero?')
+        .type('Boolean')
+
+    hero.createField('heroBackground')
```

Comment:
Do we need another field for "connector colour"?

Reply:
will be updating the background colour options in contentful.